tests/test_properties.py

```diff
@@ -0,0 +1,269 @@
+# Copyright Mozilla Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from importlib.resources import files
+from textwrap import dedent
+from unittest import TestCase
+
+from moz.l10n.properties import properties_parse, properties_serialize
+from moz.l10n.resource import Entry, Resource, Section
+
+
+class TestProperties(TestCase):
+    def testBackslashes(self):
```

Comment:
Is there are reason not to use the snake_case here and in the next test?

Reply:
No real reason, this is a remnant from having been adopted from [here](https://github.com/mozilla/compare-locales/blob/5f2e98b587d1e2193b4531aea5890e3aa510aacf/compare_locales/parsers/tests/test_properties.py#L15), which used camelCase.